Warlords of Draenor

wyldclaw opened the pull request:
A massive push to get Handynotes: Warlords of Draenor up to the same level as Handynotes: Shadowlands.

- I've included _all_ (I hope?!) rare spawns and treasures.
- I introduced a new Follower reward which could be helpful in Shadowlands content too.
- I introduced a new Profession requirement.
- Several additional achievements have been added.
- Pepe in all 4 locations
- Some simple reorganization

_This push will fail due to a luacheck. Any changes made locally to .luacheckrc are never pushed for some reason so I need to push, make changes online, and then pull. I suspect because it is on an ignore list somewhere._

Commit:
Add new Profession requirement

Commit:
Add new Follower reward

Commit:
Add new follower nodes

Commit:
Add all rare and treasure nodes, add follower nodes

Commit:
Add follower nodes, Pepe, and reorganization

Commit:
Add all rare and treasure nodes, Pepe, Steamwheedle rares, [Broke Back Precipice] nodes, and follower nodes

Commit:
Add all rare and treasure nodes and follower nodes

Commit:
Add all rare and treasure nodes, [King of the Monsters] nodes, and follower nodes

Commit:
Add all treasure nodes, Pepe, [Wingmen] nodes, and  follower nodes

Commit:
Add groups for new nodes

Commit:
Add new localization and sort old localization

Commit:
Add localization for follower types

Commit:
Update .luacheckrc

zarillion commented:
> *This push will fail due to a luacheck. Any changes made locally to .luacheckrc are never pushed for some reason so I need to push, make changes online, and then pull. I suspect because it is on an ignore list somewhere.*

The only ignore list would be here: https://github.com/zarillion/handynotes-plugins/blob/master/.gitignore

What IDE are you using to edit the project? It must be something local to your development environment. When I modify the `.luacheckrc` file in my own repo it shows with tracked unstaged changes:

```
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   .luacheckrc

no changes added to commit (use "git add" and/or "git commit -a")
```

ghost commented:
I just develop straight in Sublime Text 3. I've tried to add `.luacheckrc` a few times but it never gets added to tracked changes. I don't really know enough to fix it either :(

Commit:
Fix locale order for ruRU

Krovikan committed:
Updated Core WoD esES

Krovikan committed:
Updated FrostFire & Gorgrond WoD esES

Krovikan commented:
Please, beware with reorders.

You put in my file (esES) locations for Germany (deDE).

I reput my locations thanks to the diff of the commit.


Regards

Commit:
Fix Aeda icon

Commit:
Fix Blook node and fix [Prove Your Strength] nodes

Commit:
Add fel portal and fix Pepe label/note

Commit:
Add fel portal

Krovikan committed:
Updated Pepes & Fel Portal WoD esES

Dathwada committed:
Fix typos

Dathwada committed:
Update Reusable mana potion

Dathwada committed:
Update "in a Cave" notes

Dathwada commented on plugins/06_WarlordsOfDraenor/zones/frostfire.lua:
```diff
@@ -37,13 +51,823 @@ map.nodes[64805300] = Gorok
 
+map.nodes[62604220] = Rare({
+    id = 72156,
+    note = L['borrok_the_devourer_note'],
```
This phrase doesn't exist in the translation files.

Dathwada commented on plugins/06_WarlordsOfDraenor/zones/frostfire.lua:
```diff
@@ -37,13 +51,823 @@ map.nodes[64805300] = Gorok
+map.nodes[72203600] = Rare({
+    id = 87352,
+    quest = 37380,
+    note = L['gibblette_the_cowardly_note'],
```
This phrase is missing too.

Dathwada commented on core/requirements.lua:
```diff
@@ -107,6 +107,31 @@ end
+    local professions = {prof1, prof2, archaeology, fishing, cooking, firstAid}
+    for i = 1, #professions do
+        if professions[i] ~= nil then
+            if self.profession == GetProfessionInfo(professions[i]) then
```
GetProfessionInfo(id) returns at index 1 the translated profession name.
![](http://puu.sh/Jg8xn/ed1e313dd7.jpg)
The condition will only be true for english players.

Dathwada replied:
I just pushed an update for this.

Dathwada committed:
Update frozen slaves

Dathwada committed:
Update deDE

Krovikan committed:
Some updates WoD esES

Krovikan commented:
For this line:

L['rooby_roos_ruby_collar_note'] = 'Buy 3 {item:114835} from {npc:82432} in the basement of the |cffffff00Inn|r. Follow and feed {npc:84332} until he leaves a "treasure" on the floor.'

I found in a video for the item inside the trasesure that the Inn have a name. In Spanish is "El Percebe Salobre".

Meanwhile I was writing the above paragraph I found a video in English. In English the name of the inn is "The Briny Barnacle".

For me, the line remain as that (I write in English my Spanish translation):

L['rooby_roos_ruby_collar_note'] = 'Buy 3 {item:114835} from {npc:82432} in the basement **of the inn |cffffff00The Briny Barnacle|r**. Follow and feed {npc:84332} until he leaves a "treasure" on the floor.'

Dathwada committed:
Update profession requirement

We know that the id 11 is for archaeology.
GetProfessionInfo(self.profession) returns the translated name for us and firstAid is no more.

Commit:
Add note for Borrok, Gibblette, and Ruby Roo's Collar

Krovikan committed:
Done last updates, Dathwada and Borrok WoD esES

Krovikan commented:
In this note, really are Elemental Totems, not Stones.

L['bounty_of_the_elements_note'] = 'Use the Elemental Stones for access.'

Regards

Krovikan committed:
Updated Nagrand WoD esES

Krovikan commented:
In this line:

L['goldmane_follower_note'] = 'Kill {npc:80080} nearby to loot {item:111863} and unlock the cage to recruit him as a follower'

seems that you want recruit the {npc:80080}. I change the phrase to:

L['goldmane_follower_note'] = 'Kill {npc:80080} nearby to loot {item:111863} and **open** the cage to recruit **the prisoner** as a follower.'


Why this is in Spires of Arak section when the treasure is in Tanaan? (I know that the first part, elixir, is in Spires. But is possible put in Tanaan writing that "take a elixir in Spires" or somehow) :

L['elixir_of_shadow_sight_treasure'] = 'Elixir of Shadow Sight'
L['elixir_of_shadow_sight_note'] = 'Take to a |cffffff00Shrine to Terokk|r. This is repeatable.'


Regards

Commit:
Fix treasures and follower note

Krovikan committed:
More updates and fixes WoD esES

Krovikan committed:
Updated Shadowmoon WoD esES

Dathwada committed:
Fix Nagrand treasures

Dathwada committed:
Update deDE.lua

Krovikan committed:
fixed typo WoD esES

Commit:
Fix farm nodes for [Wingmen]

Krovikan committed:
Updated Spires of Arak WoD esES

Krovikan commented:
This line:

L['egg_of_varasha_treasure'] = 'Egg of Varasha'

should be, according to wowhead:

L['egg_of_varasha_treasure'] = 'Varasha\'s Egg'

"The Egg of Varasha" is a vignette quest.

"Varasha\'s Egg" is a container and interactive object.

Regards

adavak committed:
Update zhCN.lua

adavak committed:
Update zhCN.lua

adavak committed:
typo

adavak committed:
Update zhCN.lua

adavak committed:
Update zhCN.lua

Krovikan committed:
Updated Talador WoD esES

Krovikan commented:
Wrong name for string:

L['isaaris_cache_note'] = 'Rescue 4 draenei trapped in spider webs, then Isaari\'s Cache will spawn here'
L['**isarris**_cache_treasure'] = 'Isaari\'s Cache'

The second string should be named:
'**isaaris**_cache_treasure'

Regards
